### PR TITLE
Add Spago test config

### PR DESCRIPTION
**Pull request by triallax**
See https://github.com/sharkdp/purescript-quantities/pull/46#issuecomment-875014553 for context.

### CODE REVIEW

File: spago.dhall [hunk 1]
```diff
@@ -17,11 +16,10 @@ You can edit this file as you like.
   , "pairs"
   , "prelude"
   , "psci-support"
```
Comment (triallax) [inline spago.dhall:18]: Should I put this into `test.dhall`? It's not exactly a test dependency though.

Comment (sharkdp) [reply spago.dhall:18]: Hm, let's leave it for now.

**Review by sharkdp — APPROVED** [review]
Thank you!

**Comment by triallax** [comment]
Mind publishing a new release?

**Comment by sharkdp** [comment]
done! I hope the saved release log is still up to date.

**Pull request merged**